setup.cfg

```diff
@@ -37,3 +37,7 @@ xfail_strict=true
 
 [metadata]
 license_files = docs/source/license.rst
+
+[mypy]
+ignore_missing_imports = True
+disable_error_code = attr-defined
```

Comment:
Note for later reference: these are needed as Cython part is not typed at all and Python code calling APIs written in Cython raises an error.